pages/common/git-delta.md

```diff
@@ -0,0 +1,9 @@
+# git delta
+
+ > Lists files that differ from another branch.
+ > Part of git-extras.
+ > More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-delta>
+
+ - List files from {{currbranch}} that differ with {{main}}:
```

Comment:
Did you mean to add `currbranch` to the example below, or this?
```suggestion
 - List files from the current branch that differ with the `main` branch:
```

Reply:
```suggestion
 - List files from current checked out branch that differ from main:
```

Reply:
They're both almost identical — @CleanMachine1 yours doesn't backtick `main`, but it should because `main` is a name here rather than an adjective.

Reply:
Why did you edit blue's suggestion...